src/breakpoints/index.ts

```diff
@@ -81,7 +80,6 @@ export namespace Breakpoints {
     }
 
     breakpointsChanged = new Signal<this, IBreakpoint[]>(this);
-    clearedBreakpoints = new Signal<this, SessionTypes | null>(this);
```

Comment:
`clearedBreakpoints` is the same as `breakpointsChanged` for the special case of `[]`.